plugin.py

```diff
@@ -271,7 +264,7 @@ def pulls(self, irc, msg, args, slug):
 
         try:
             results = sum([
-                list(self.yield_github_pulls(username, repo)) for repo in repos
+                list(self.yield_github_pulls(username, r)) for r in repos
```

Comment:
I would expect pep8 to like better the old variable name (or maybe it's pylint that complains about this)